code/modules/clothing/glasses/glasses.dm

```diff
@@ -69,6 +69,19 @@
 /obj/item/clothing/glasses/meson/prescription
 	prescription = 1
 
+/obj/item/clothing/glasses/meson/gar
+	name = "gar mesons"
+	icon_state = "garm"
+	item_state = "garm"
+	desc = "Do the impossible, see the invisible!"
+	force = 10
+	throwforce = 10
+	throw_speed = 4
+	attack_verb = list("sliced")
+	hitsound = 'sound/weapons/bladeslice.ogg'
+	sharp = 1
+	edge = 1

```

Comment:
These are actually an okay melee weapon (slightly better than toolbox), that's hilarious.